core/src/main/scala/filodb.core/store/IngestionConfig.scala

```diff
@@ -39,7 +39,10 @@ final case class StoreConfig(flushInterval: FiniteDuration,
                              // filters on ingested records to log in detail
                              traceFilters: Map[String, String],
                              maxDataPerShardQuery: Long,
-                             meteringEnabled: Boolean) {
+                             meteringEnabled: Boolean,
+                             // approx data resolution, used for estimating the size of data to be scanned for
+                             // answering queries
+                             resolution: Int) {
```

Comment:
Need unit as "ms or s". Could rename as `estimatedResolution`

Reply:
Name needs to also clarify that it is not downsample resolution

Reply:
`estimatedIngestResolution` ? 😬

Reply:
ack, changes the field name.